frontend/app/player/MessageDistributor/managers/AssistManager.ts

```diff
@@ -159,8 +166,16 @@ export default class AssistManager {
           }
         }
 
+        if (this.timeTravelJump) {
+          return;
+        }
+
         for (let msg = reader.readNext();msg !== null;msg = reader.readNext()) {
           //@ts-ignore
+          if (this.jumped) {
+            msg.time = this.md.getLastRecordedMessageTime() + msg.time
```

Comment:
`msg.time` here is set by `MStreamReader` where the `startTimestamp` is the first received message.

This will cause a bug: watch live session for several minutes, then go back - the last message from the file will be a few seconds ago, and you will have a gap after that because newly received messages have  time of 2+ minutes you watched live.

Solution:
Update reader  with new start time (when MessageDistributor updates sessionStart) instead.

Reply:
still here